tests/Functional/QueryTest.php

```diff
@@ -20,7 +20,6 @@ public function testCorrectlyIdentifiesR2D2AsTheHeroOfTheStarWarsSaga()
         /** @noinspection PhpUnhandledExceptionInspection */
         $result = graphql(starWarsSchema(), $query);
 
-        $this->markTestIncomplete('INCOMPLETE: We cannot yet resolve data for interface types.');
```

Comment:
@hungneox Thanks, for fixing this test case. I'll continue porting the rest of the query tests soon.